go/contacts/tasks.py

```diff
@@ -33,10 +33,13 @@ def delete_group(account_key, group_key):
     # We do this one at a time because we're already saving them one at a time
     # and the boilerplate for fetching batches without having them all sit in
     # memory is ugly.
-    for contact_key in group.backlinks.contacts():
-        contact = contact_store.get_contact_by_key(contact_key)
-        contact.groups.remove(group)
-        contact.save()
+    contacts_page = group.backlinks.contact_keys()
+    while contacts_page is not None:
+        for contact_key in contacts_page:
+            contact = contact_store.get_contact_by_key(contact_key)
+            contact.groups.remove(group)
+            contact.save()
+        contacts_page = contacts_page.next_page()
```

Comment:
If it is as complicated using something like `PausingDeferredQueue`, then agreed that it is out of scope. I'm not understanding why we'd have to use something like that instead of just moving this boilerplate into a method or function though. Wouldn't doing something like this be identical to what we are doing now, except without the repetition?

```python
def all(page):
    while page is not None:
        for key in page:
            yield key
        page = page.next_page()    
```